08-group_average_sensors.py

```diff
@@ -0,0 +1,37 @@
+"""
+=====================================
+11. Group average at the sensor level
+=====================================
+
+The EEG-channel data are averaged for group averages.
+"""
+
+import os.path as op
+
+import mne
+
+from library.config import meg_dir, l_freq, exclude_subjects
+
+all_evokeds = [list() for _ in range(7)]  # Container for all the categories
+
+for run in range(1, 20):
```

Comment:
this needs to be put inside a function like the other scripts. Something like:
```py
def run_maxwell_filter(subject):
    bla bla 
```

and called at the bottom like:
```py
parallel, run_func, _ = parallel_func(run_maxwell_filter, n_jobs=N_JOBS)
parallel(run_func(subject) for subject in subjects_list)
```